db/seeds.rb

```diff
@@ -1 +1,10 @@
+users = [{ :first_name => "Kermit", :last_name => "Frog"},
+{:first_name => "Kulfon", :last_name => "Maly"},
```

Comment:
we should use new notation for hashes, which is `first_name: "Kulfon"` instead of `:first_name => "Kulfon"`, moreover I think it looks better when you use spaces like in the line above